pkgs/tools/X11/wpgtk/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, python3, python36Packages, fetchFromGitHub, pywal, feh, libxslt, imagemagick,
+  gobjectIntrospection, gtk3, wrapGAppsHook, gnome3 }:
+
+python36Packages.buildPythonApplication rec {
+  name = "${pname}-${version}";
+  pname = "wpgtk";
+  version = "5.7.4";
+
+  src = fetchFromGitHub {
+    owner = "deviantfero";
+    repo = "wpgtk";
+    rev = "${version}";
+    sha256 = "0c0kmc18lbr7nk3hh44hai9z06lfsgwxnjdv02hpjwrxg40zh726";
+  };
+
+  pythonEnv = python3.withPackages(ps: with ps; [ pygobject3 pillow pip]);
```

Comment:
You should not use both `python` and `pythonPackages` in a single package – it can lead to issues when user overrides one but not the other.

Reply:
Also, since this package uses `buildPythonApplication`, adding the libraries to `buildInputs` or `pythonPath` attribute should be enough.